docs/concepts/cluster-administration/addons.md

```diff
@@ -14,7 +14,7 @@ Add-ons in each section are sorted alphabetically - the ordering does not imply
 
 
 * [ACI](https://www.github.com/noironetworks/aci-containers) provides integrated container networking and network security with Cisco ACI.
-* [Calico](http://docs.projectcalico.org/latest/getting-started/kubernetes/installation/hosted/) is a secure L3 networking and network policy provider.
+* [Calico](https://docs.projectcalico.org/v3.1/getting-started/kubernetes/) is a secure L3 networking and network policy provider.
```

Comment:
how about changing this to "https://docs.projectcalico.org/latest" which is more stable than the specific url path while avoid being trapped to version specifics?

Reply:
I find that this https://docs.projectcalico.org/latest/getting-started/kubernetes/ is valid，and will redirect to current version. how about this one. I suggest this url should direct to kubernetes install page.

Reply:
Fine in this case. I was suggesting a shorter path because the change of the URL path rings an alarm to me.

Reply:
thanks！